Changes

```diff
@@ -1,5 +1,60 @@
 Release history of Zonemaster
 
+v2018.2 2019-01-27 Pre-release
+
+Status
+- This is a pre-release version not fully tested on all supported
+  OS's and Perl versions. This version will not be available on
+  CPAN.
+
+Changes Zonemaster repository:
+- All link references on Github now to zonemaster/zonemaster instead
+  of old dotse/zonemaster #637
+- Updated specification of test case Nameserver01 (#625, #639)
+- Updated specification of test case DNSSEC05 #626
+- Updated specification of test case Delegation01 (#609, #638)
+- Updated specification of test case Delegation02 (#610)
+- Added new test case Consisteny06 (#619)
+- Updated specification of test case Consistency03 (#618)
+- Updated specification of test case Consistency02 (#616)
+- Updated specification of test case Consistency01 (#615)
+- Added new test case Nameserver11 (#652, #661)
+- Added new test case Nameserver12 (#658, #662)
+- Added new test case Nameserver13 (#653)
+- Added new test case Nameserver14 (#654)
+- Added new test case Nameserver14 (#650)
```

Comment:
Added new test case Nameserver14 (#650, #654)

Reply:
Fixed. Should be Namesever10 (#650)